packages/ui/Scroller/ScrollerBasic.js

```diff
@@ -115,7 +115,30 @@ class ScrollerBasic extends Component {
 		return x;
 	}
 
-	// for Scrollable
+	// scrollMode 'native' only
+	scrollToPosition (x, y) {
+		this.props.scrollContentRef.current.scrollTo(this.getRtlPositionX(x), y);
+	}
+
+	// scrollMode 'native' only
+	didScroll (x, y) {
+		this.scrollPos.left = x;
+		this.scrollPos.top = y;
+	}
+
+	// scrollMode 'translate' only
+	setScrollPositionTarget (x, y) {
```

Comment:
I don't think we need this function in `Scroller`.
There is no reference case for `scrollPositionTarget` from themed `Scroller`.
Why did you add this?

Reply:
Oh, my mistake. I should be removed.